tasks/reference.js

```diff
@@ -67,8 +69,8 @@ gulp.task('reference', ['clean-dist', 'less'], function() {
                 _.forEach(data.paths, function(path, pathUri) {
                     _.forEach(path, function(operation, verb) {
                         // This is where we filter the endpoints depending on their availability in the PIM versions
-                        if (((version === '1.7') && (operation['x-versions'][0] === "1.7")) ||
-                             (version === '2.0' && (operation['x-versions'][0] === "2.0" || operation['x-versions'][1] === "2.0")) || version === '2.1') {
+                        if (((version === '1.7') && (operation['x-versions'][0] === '1.7')) ||
```

Comment:
we will reach the limit of hardcoding things very fast :)

Reply:
Yup. We need to find time to rework this.